sparkle/package.json

```diff
@@ -112,6 +112,7 @@
     "react-confetti": "^6.1.0",
     "react-dropdown-menu": "^0.0.2",
     "react-markdown": "^9.0.1",
+    "shadcn-ui": "^0.9.3",
```

Comment:
OOC: how come do we need this as a dependency?

Reply:
@Duncid I guess that you'll be able to answer